src/components/editor/List.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactList from 'react-list';
+import { Link } from 'react-router';
+
+export default class List extends React.Component {
+  constructor(props) {
+    super(props);
+    this.renderItem = this.renderItem.bind(this);
+  }
+  renderItem(index, key) {
+    index = this.props.indexConverter[index];
+    return (
+      <div key={key}>
+        <Link to={this.props.link + this.props.data[index].slug + "/"}>{this.props.data[index].title}</Link>
+      </div>
+    );
+  }
+  render() {
+    return (
+      <div style={this.props.style}>
+        {this.props.name ? this.props.name : null}
+        <div style={{overflow: 'auto', maxHeight: this.props.maxHeight, maxWidth: this.props.maxWidth}}>
+          <ReactList
```

Comment:
It can be scrolling, just user overflow:hidden (I think). The react-list you imported specifically will render and un-render elements as it is scrolled, and it's actually quite a heavy class. If you just want it for visual purposes, just use regular html, and you can make subscrollable boxes with overflow.